ha/note.c

```diff
@@ -168,13 +220,24 @@ static void ha_state_accept(struct m0_confc         *confc,
 	cache = &confc->cc_cache;
 	m0_conf_cache_lock(cache);
 	for (i = 0; i < note->nv_nr; ++i) {
-		obj = m0_conf_cache_lookup(cache, &note->nv_note[i].no_id);
+		obj = m0_conf_cache_lookup_dynamic(cache, &note->nv_note[i].no_id);
 		M0_LOG(M0_DEBUG, "nv_note[%d]=(no_id="FID_F" no_state=%"PRIu32
 		       ") obj=%p obj->co_status=%d", i,
 		       FID_P(&note->nv_note[i].no_id),
 		       note->nv_note[i].no_state,
 		       obj, obj == NULL ? -1 : obj->co_status);
-		if (obj != NULL && obj->co_status == M0_CS_READY) {
+		if ((obj != NULL) &&
+		    (m0_fid_tget(&note->nv_note[i].no_id) == 'r') &&
+		     !cache->ca_is_phony &&
+		     (!m0_fid_eq(&obj->co_id, &note->nv_note[i].no_id))) {
+			/*
+			 * We have received new Dynamic FID. Adding
```

Comment:
Chang this to,

 * Received new Dynamic configuration object, adding
*  this to configuration cache.

Reply:
Done.